src/main/java/org/apache/maven/plugins/jlink/AbstractJLinkToolchainExecutor.java

```diff
@@ -0,0 +1,204 @@
+package org.apache.maven.plugins.jlink;
+
+/*
```

Comment:
copyright usually goes above package

Reply:
I know, but the checkstyle prevents this. I am not *also* going to change the checkstyle configuration in this PR. It's already very big.

Reply:
Actually, you can figure out which checks are being enforced this way:

```
(user@host)[~/git/apache/maven-jlink-plugin]$ mvn help:effective-pom
[INFO] Scanning for projects...
[INFO] 
[INFO] ------------< org.apache.maven.plugins:maven-jlink-plugin >-------------
[INFO] Building Apache Maven JLink Plugin 3.0.1-SNAPSHOT
[INFO] ----------------------------[ maven-plugin ]----------------------------
[INFO] 
[INFO] --- maven-help-plugin:3.2.0:effective-pom (default-cli) @ maven-jlink-plugin ---
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/maven-model/3.6.1/maven-model-3.6.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/maven-model/3.6.1/maven-model-3.6.1.pom (4.0 kB at 5.7 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/maven/3.6.1/maven-3.6.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/maven/3.6.1/maven-3.6.1.pom (24 kB at 523 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/maven-model/3.6.1/maven-model-3.6.1.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/maven-model/3.6.1/maven-model-3.6.1.jar (186 kB at 2.2 MB/s)
[INFO] 
Effective POMs, after inheritance, interpolation, and profiles are applied:

[…]
        <plugin>
          <artifactId>maven-checkstyle-plugin</artifactId>
          <version>3.0.0</version>
          <dependencies>
            <dependency>
              <groupId>org.apache.maven.shared</groupId>
              <artifactId>maven-shared-resources</artifactId>
              <version>2</version>
              <scope>compile</scope>
            </dependency>
          </dependencies>
          <configuration>
            <configLocation>config/maven_checks.xml</configLocation>
            <headerLocation>config/maven-header.txt</headerLocation>
[…]
```

Then: https://github.com/apache/maven-shared-resources/blob/master/src/main/resources/config/maven-header.txt

HTH